gravitee-apim-console-webui/src/management/application/details/subscriptions/creation/application-subscription-creation-dialog.models.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { FormControl, FormGroup } from '@angular/forms';
+
+import { Api, CreateSubscription, Plan } from '../../../../../entities/management-api-v2';
+import { ApiKeyMode } from '../../../../../entities/application/Application';
+
+export type SubscriptionCreationForm = FormGroup<{
```

Comment:
WDYT to add form Type where we instantiate FormGroup ? to have this code close to this uses 🤷‍♂️